### ISSUE TITLE
Does PicoFlasher support writing timefile to Coolrunner/Matrix/Ace glitch board?

### ISSUE THREAD

**Issue by shooter556** [X360Tools/PicoFlasher#13]
If so, how to connect the wires?
Thank you.

**Comment by hzqxwvd596** [comment]
No.

**Comment by nikdiklic** [comment]
With the new RGH 3 method, most of the aforementioned glitch chips have been made obsolete. Please look into the RGH3 process and if your console is vulnerable (Falcon, Jasper, Trinity, Corona)

**Comment by shooter556** [comment]
> With the new RGH 3 method, most of the aforementioned glitch chips have been made obsolete. Please look into the RGH3 process and if your console is vulnerable (Falcon, Jasper, Trinity, Corona)

Thanks for your reply, that makes sense.
I have tried RGH3, it works perfectly on slim/slimE consoles for sure. (Almost all boots instantly)
But when I tried RGH3 on phat consoles, it doesn't look so nice. Boot time may need some luck to be perfect. Sometimes it boots instantly, but most of time, 10-30 seconds will be required.
So considering programming glitch board may still be needed until RGH3 is perfect on phats, I'm just wondering is there any possibility/plan to add such function to picoflasher.
